cfgov/login/tests/test_viewsets.py

```diff
@@ -0,0 +1,27 @@
+from django.apps import apps
+from django.test import TestCase
+
+from wagtail.users.wagtail_hooks import get_viewset_cls
+
+from login.forms import UserCreationForm, UserEditForm
+
+
+class UserViewSetTestCase(TestCase):
+    """Test that the wagtailusers app config loads our UserViewSet.
+
+    This tests both our UserViewSet class (which just returns our custom forms)
+    as well as our LoginUsersAppConfig in apps.py."""
```

Comment:
[Per PEP8](https://peps.python.org/pep-0008/#documentation-strings), "the """ that ends a multiline docstring should be on a line by itself".

```suggestion
    as well as our LoginUsersAppConfig in apps.py.
    """
```